vagrant/lib/forklift/box_distributor.rb

```diff
@@ -287,12 +290,16 @@ def configure_rackspace(machine, box)
 
     def configure_openstack_provider(machine, box)
       machine.vm.provider :openstack do |p, override|
-        override.vm.box        = nil
-        override.ssh.username  = 'root'
+        override.vm.box = nil
+        if box.fetch('sync_type', 'disabled')
```

Comment:
Perhaps it'd be better to mirror what `configure_libvirt` does:
```
override.vm.synced_folder '.', '/vagrant', type: @settings['sync_type'], rsync__args: ['--max-size=100m']
```

This could be enhanced (for both) by using `box.fetch('sync_type', @settings['sync_type'])`

Reply:
Yea I like your idea, but actually I need to disable rsync by disable option:
`override.vm.synced_folder '.', '/vagrant', disabled: true` 
but `type: @settings['sync_type']` is only giving me an option to set another type of synced folders. So I create in boxes `sync_type: disabled, in commit history you can see I used `sync_disabled: true` but some of you have a good point it should be sync_type but now sync_type  make this confusion too.